### PR TITLE
determine if new report needs to be generated pending on latest updat…

### PR DESCRIPTION
**Pull request by itaymmguardicore**
# Feature / Fixes
> Determine whether to generate new attack report based on latest monkey modifytime

* [X] Have you added an explanation of what your changes do and why you'd like to include them?
* [X] Have you successfully tested your changes locally?

### CODE REVIEW

**Comment by danielguardicore** [comment]
Can we do this over the new DAL?

File: monkey/monkey_island/cc/services/attack/attack_report.py [hunk 1]
```diff
@@ -25,7 +26,13 @@ def generate_new_report():
         Generates new report based on telemetries, replaces old report in db with new one.
         :return: Report object
         """
-        report = {'techniques': {}, 'latest_telem_time': AttackReportService.get_latest_attack_telem_time(), 'name': REPORT_NAME}
+        report =\
+            {
+                'techniques': {},
+                'meta': {'latest_monkey_modifytime': NodeService.get_latest_modified_monkey()[0]['modifytime']},
```
Comment (ShayNehmad) [inline monkey/monkey_island/cc/services/attack/attack_report.py]: Code duplication between here and line 53. Perhaps it's a good opportunity to move this function + a wrapper for getting just the time to the models module and implement using mongoengine instead? :)

**Comment by danielguardicore** [comment]
Looks good to me, lets solve the conflict and merge.

**Comment by ShayNehmad** [comment]
❤ for using the Monkey data model

**Pull request merged**